1/test/run

```diff
@@ -22,13 +22,8 @@ function cleanup() {
     echo "Stopping and removing container $CONTAINER..."
     docker stop $CONTAINER
     exit_status=$(docker inspect -f '{{.State.ExitCode}}' $CONTAINER)
-    # jenkins "normally" exits with code 143:
-    # http://jenkins-ci.361315.n4.nabble.com/Jenkins-exit-code-143-due-to-Unexpected-termination-of-the-channel-td4652347.html
-    # tl;dr: 128+signal=return code, and "15" equates to SIGTERM
-    if [ "$exit_status" != "143" ]; then
-      echo "Dumping logs for $CONTAINER"
-      docker logs $CONTAINER
-    fi
+    echo "Dumping logs for $CONTAINER"
+    [[ "${exit_status}" != "0" ]] && docker logs $CONTAINER
```

Comment:
@bparees hrm, I would rather remove the check and display all container output (it really helps debugging).

On the other hand, can we fix Jenkins to return 0 when sigtermed? I guess it won't be showing well in the console or in CloudForms when the container exit will not be graceful. However, I don't know how to do it 'smart' way with exec.